pyre/extra_tests/snippets/stdlib_native_contracts.py

```
@@ -307,6 +359,12 @@ class StringSubclass(str):
     assert refusal(lambda: compile("a\0b", "<s>", "exec"), SyntaxError) == (
         "source code string cannot contain null bytes"
     )
+    try:
+        os.symlink("/tmp", "/tmp")
```

Ruff (0.16.3)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)